.pre-commit-config.yaml

```diff
@@ -42,7 +42,7 @@ repos:
           - packaging
           - py>=1.9.0
   - repo: https://github.com/pre-commit/mirrors-pylint
-    rev: v2.7.2
+    rev: v3.0.0a1
```

Comment:
@asottile can we opt out of auto-update from alpha releases?

Reply:
they're just tags, pre-commit doesn't know the difference

Reply:
Could try to parse it as semver and if it manages and it's alhpa/beta ignore it? I don't want to use alpha builds just yet in my projects.

Reply:
no sorry, that's a slippery slope I don't want to get into.  tags are just strings

Reply:
(there's also no reasonable way to say "git, please give me the latest tag, except ignore this particular tag that parses as a version that's ...")

Reply:
:-/ banlist?

Reply:
even with a banlist, there isn't a way to make git do what you want it to do

and a banlist is unmaintainable

Reply:
I see. This makes pre-commit.ci no longer desirable to me because it creates PRs with version bumps I don't want to support. The only way to auto-update is to intervene manually...